library/psa_crypto.c

```diff
@@ -3876,6 +3734,47 @@ psa_status_t psa_aead_decrypt( mbedtls_svc_key_id_t key,
     return( status );
 }
 
+static psa_status_t psa_validate_tag_length( psa_aead_operation_t *operation,
+                                             psa_algorithm_t alg ) {
+    uint8_t tag_len = 0;
+    if( psa_driver_get_tag_len( operation, &tag_len ) != PSA_SUCCESS )
+    {
+        return( PSA_ERROR_INVALID_ARGUMENT );
+    }
+
+    switch( PSA_ALG_AEAD_WITH_SHORTENED_TAG( alg, 0 ) )
+    {
+#if defined(PSA_WANT_ALG_CCM)
```

Comment:
We need a commit message explaining why are changing which defines we use (more detailed explanation than makes it build).

And explaining why it's noup and not fromlist.

https://github.com/nrfconnect/sdk-mbedtls/pull/15/commits/1b53560ea805b8abdbfd13dc1cc24432d72e0ba6

Reply:
The same for

"14e0cd499 * [nrf toup] Fix RSA dependency on PK write when enabling PSA crypto"